lib/classes/ConfigSchemaHandler/regexpKeyword.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+// Implementation has been heavily inspired by `ajv-keywords` implementation from:
+// https://github.com/ajv-validator/ajv-keywords/blob/9656614be0104fe71d229459a5217ccac11958a6/src/definitions/regexp.ts
```

Comment:
Might be worth adding `Copyright (c) 2016 Evgeny Poberezkin` in that comment block (with a link to https://github.com/ajv-validator/ajv-keywords/blob/master/LICENSE) to comply with the MIT license: https://github.com/ajv-validator/ajv-keywords/blob/master/LICENSE

Reply:
good point, added that to the commit